crates/sui-framework/tests/delegation_tests.move

```diff
@@ -21,6 +22,79 @@ module sui::delegation_tests {
     const DELEGATOR_ADDR_1: address = @0x42;
     const DELEGATOR_ADDR_2: address = @0x43;
 
+    #[test]
+    fun test_split_join_staked_sui() {
+        let scenario_val = test_scenario::begin(DELEGATOR_ADDR_1);
+        let scenario = &mut scenario_val;
+        // All this is just to generate a dummy StakedSui object to split and join later
+        set_up_sui_system_state(scenario);
+        test_scenario::next_tx(scenario, DELEGATOR_ADDR_1);
+        {
+            let system_state = test_scenario::take_shared<SuiSystemState>(scenario);
+            let system_state_mut_ref = &mut system_state;
+            let ctx = test_scenario::ctx(scenario);
+            sui_system::request_add_delegation(
+                system_state_mut_ref, coin::mint_for_testing(60, ctx), VALIDATOR_ADDR_1, ctx);
+            test_scenario::return_shared(system_state);
+        };
```

Comment:
You can use `governance_test_utils::delegate_to` as an easier way to write this transaction if you want:
```suggestion
        governance_test_utils::delegate_to(DELEGATOR_ADDR_1, VALIDATOR_ADDR_1, 60, scenario);
```